### PR TITLE
Add AA code to position text, e.g. 25Y

### PR DESCRIPTION
**Pull request by fredludlow**
Hi there, not sure if you'll want to merge this (or if you'd rather have it as configurable option) but a minor request from a user to have position include the sequence text (e.g. 25Y rather than just 25).

### CODE REVIEW

File: src/feature-viewer.js [hunk 1]
```diff
@@ -1554,8 +1554,10 @@ var FeatureViewer = (function () {
                 .attr("in", "SourceGraphic");
 
             svgContainer.on('mousemove', function () {
-                var absoluteMousePos = SVGOptions.brushActive ? d3.mouse(d3.select(".background").node()) : d3.mouse(svgContainer.node());; 
-                $(div + " #zoomPosition").text(Math.round(scalingPosition(absoluteMousePos[0])));
+                var absoluteMousePos = SVGOptions.brushActive ? d3.mouse(d3.select(".background").node()) : d3.mouse(svgContainer.node());;          
+                var pos = Math.round(scalingPosition(absoluteMousePos[0]));
+                pos += sequence[pos] || "";
```
Comment (MatSchaeff) [inline src/feature-viewer.js]: Shouldn't it be ```pos += sequence[pos-1] || ""; ``` ?
the scale range begins at 1, but the sequence first index is 0.

**Comment by MatSchaeff** [comment]
Hi Fred,

Thank you for your contribution !
I left you a comment on the code about the ```seq[pos] ```.
example

If you can fix this, and i will merge your pull-request :)

Cheers,
Mathieu

**Comment by fredludlow** [comment]
Ah, so this stems from my misunderstanding of the offset parameter in the constructor. I had assumed I could use that as a sort of "initial zoom" argument (and had been incorrectly passing {start:0, end: length-1}). Turns out that with offset.start = 0 you get some slightly unexpected behaviour that cancels out my off-by-one error!

The change you suggested does indeed fix it (for offset.start>0) so I'll update my PR. 

This means I'll need to rethink setting the zoom programatically, but I guess from your comment to #4 that's something that's in the pipeline?

**Comment by MatSchaeff** [comment]
Oh ok ! 
If an offset is not specified within the options, the feature-viewer will automatically render a instance with ```var offset = {start:1,end:sequence.length};```, that's the "initial zoom".
I should have specified in the doc that the offset.start need to be >0.
I think i will add a check, and if needed, change the offset.start and add a console.warning to inform that the offset.start have been reset to 1.

I've got a lot of side projects going on, but i think i will deliver #4 this week :)
Keep in touch,

Cheers,
Mathieu

**Pull request merged**